2021-05-09 01:14:28,474 INFO    MainThread:24264 [wandb_setup.py:_flush():69] setting env: {}
2021-05-09 01:14:28,474 INFO    MainThread:24264 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 01:14:28,474 INFO    MainThread:24264 [wandb_init.py:_log_setup():336] Logging user logs to /home/trenton-windows/repos/spicy-dann/wandb/run-20210509_011428-2rwwe2sx/logs/debug.log
2021-05-09 01:14:28,474 INFO    MainThread:24264 [wandb_init.py:_log_setup():337] Logging internal logs to /home/trenton-windows/repos/spicy-dann/wandb/run-20210509_011428-2rwwe2sx/logs/debug-internal.log
2021-05-09 01:14:28,474 INFO    MainThread:24264 [wandb_init.py:init():369] calling init triggers
2021-05-09 01:14:28,474 INFO    MainThread:24264 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-09 01:14:28,475 INFO    MainThread:24264 [wandb_init.py:init():418] starting backend
2021-05-09 01:14:28,475 INFO    MainThread:24264 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-09 01:14:28,505 INFO    MainThread:24264 [backend.py:ensure_launched():132] starting backend process...
2021-05-09 01:14:28,521 INFO    MainThread:24264 [backend.py:ensure_launched():137] started backend process with pid: 24292
2021-05-09 01:14:28,522 INFO    MainThread:24264 [wandb_init.py:init():423] backend started and connected
2021-05-09 01:14:28,525 INFO    MainThread:24264 [wandb_init.py:init():465] updated telemetry
2021-05-09 01:14:28,525 INFO    MainThread:24264 [wandb_init.py:init():484] communicating current version
2021-05-09 01:14:29,623 INFO    MainThread:24264 [wandb_init.py:init():489] got version response 
2021-05-09 01:14:29,623 INFO    MainThread:24264 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 01:14:29,796 INFO    MainThread:24264 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 01:14:33,155 INFO    MainThread:24264 [wandb_run.py:_console_start():1565] atexit reg
2021-05-09 01:14:33,156 INFO    MainThread:24264 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-09 01:14:33,162 INFO    MainThread:24264 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-09 01:14:33,169 INFO    MainThread:24264 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-09 01:14:33,169 INFO    MainThread:24264 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 01:14:33,173 INFO    MainThread:24264 [wandb_run.py:_config_callback():787] config_cb None None {'dataset': 'camelyon17', 'data_root': './WILDS', 'model_name': 'resnet18', 'batch_size': 3, 'n_epochs': 1, 'get_train_metrics': True, 'save_every': 5, 'run_name': 'dev-small', 'max_val_batches': 2}
2021-05-09 01:14:46,133 INFO    MainThread:24264 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-09 01:14:46,145 INFO    MainThread:24264 [wandb_run.py:_restore():1507] restore
2021-05-09 01:14:48,156 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 2
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 43797
  total_bytes: 43797
}

2021-05-09 01:14:48,398 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 2
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 43797
  total_bytes: 43797
}

2021-05-09 01:14:48,814 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 43797
  total_bytes: 47704
}

2021-05-09 01:14:48,994 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 46474
  total_bytes: 47704
}

2021-05-09 01:14:49,096 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 47704
  total_bytes: 47704
}

2021-05-09 01:14:49,198 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 47704
  total_bytes: 47704
}

2021-05-09 01:14:49,301 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 47704
  total_bytes: 47704
}

2021-05-09 01:14:49,403 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 47704
  total_bytes: 47704
}

2021-05-09 01:14:49,506 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 7
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 47704
  total_bytes: 47704
}

2021-05-09 01:14:50,628 INFO    MainThread:24264 [wandb_run.py:_show_files():1879] logging synced files
